builder/oracle/classic/config.go

```diff
@@ -0,0 +1,67 @@
+package classic
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/helper/communicator"
+	"github.com/hashicorp/packer/helper/config"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/template/interpolate"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+	Comm                communicator.Config `mapstructure:",squash"`
+
+	Access *AccessConfig
```

Comment:
should either move access config to this struct or remove `AccessConfig`.

Reply:
Pulled it. I don't think we were using it.